src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -319,21 +713,40 @@ public void binaryContentContentTypeAndType(final String uri) {
                         "type of the external content, if provided. Any Content-Type header in the request should be " +
                         "ignored.",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final Headers headers = new Headers(
+                new Header(LINK, externalContentHeader(externalUri, "proxy", "text/special")),
+                new Header(SLUG, info.getId()));
+        final String location = getLocation(doPost(uri, headers));
+
+
+        // Type attribute supersedes Content-Type from external server
+        doHead(location).then()
+                .header("Content-Type", "text/special");
     }
 
     /**
      * 3.9-F-6 Content-Type and no type
      *
      * @param uri of base container of Fedora server
      */
-    @Test(groups = {"SHOULD"})
-    @Parameters({"param1"})
+    @Test(groups = { "SHOULD" })
+    @Parameters({ "param1" })
     public void binaryContentContentTypeAndNoType(final String uri) {
         final TestInfo info = setupTest("3.9-F-6", "binaryContentContentTypeAndNoType",
                 "Fedora servers must use the value of the type attribute in the external content link as the media " +
                         "type of the external content, if provided. Any Content-Type header in the request should be " +
                         "ignored.",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        // Providing external content which does not return Content-Type
+        final Headers headers = new Headers(
+                new Header(LINK, externalContentHeader(externalUriWithNoType, "proxy", "text/special")),
+                new Header(SLUG, info.getId()));
```

Comment:
Please avoid using the `Slug` header where possible as support for it is not required AFAIK.